### ISSUE TITLE
Store results somewhere

### ISSUE THREAD

**Issue by tivac** [developit/esbench#3]
jsperf.com used https://www.browserscope.org/, maybe there's something else you'd prefer. Either way having the results be aggregated & stored w/ the tests instead of manually would be wonderful.

**Comment by developit** [comment]
Yes! @trueadm was asking about this too. I'm likely going to avoid BrowserScope since I had tried with then previously and the uptime was just not good enough.  

Also: since ESBench is all API-driven, extending the API to support storing and aggregating tests would mean we could also do the same thing for people using Benchmark.js in CI! It would making tracking performance over time really simple.
